### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by for-ashraf**
In this project, I have incorporated all the required features (as far as my knowledge is concerned).
A video that is a short description of the work has been uploaded on Loom (https://www.loom.com/share/f5f47479af9e422997a176ee17deecdf)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+-Javascript
+
+
+## Live Demo
+
+[Live Demo Link]
+
+## Presentation of Project
+
+[Loom Video Link]
```
Comment (keroloussamy) [inline README.md]: - [ ] Kindly update your links here, the live demo link, and the deploy link for your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+              <b>Address:</b> Main Campus <br />
+              Allama Iqbal Open University<br />
+              H-8/4, Islamabad
+            </p>
+          </div>
+        </div>
+        <p class="rights"><i class="far fa-copyright"></i>2022. All rights reserved</p>
+      </footer>
+      <script src="./js/script.js"></script>
+    </body>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly the name of your file is `scripts.js` with s but, you are writing it wrong kindly fix it.
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+              reprehenderit labore blanditiis temporibus modi.
+            </p>
+          </div>
+          <div class="infos">
+            <h3 class="date light-black heading-medium">
+              2022.07.19(Tue) ~ 21(Thu)
+            </h3>
+            <p class="location light-black heading-small">
+              @Conference Hall, Main Campus, AIOU Islamabad
+            </p>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly make your paragraph on the left a little bit and give it padding from left and right to match the original design.
- [ ] Kindly do the same for the bold text, make it on the left a little bit and give it padding from the bottom.

Your design
image

 Original design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+          <h3 class="item item-header dark-orange">Ignite</h3>
+          <p class="item item-content light-grey">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+            libero sit in necessitatibus, ad sequi?
+          </p>
+        </article>
+      </div>
+      <button type="button" name="join-us" class="btn main-btn"> Join the Program </button>
+      <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+    </section>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly the background for this section is dark, kindly change it.
- [ ] Kindly remove this space, there's no need for it, to match the layout of the original design.

Your design
image

Original design
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            </div>
+            <div class="info">
+              <h3 class="date light-black heading-medium">
+                2022.07.19(Tue) ~ 21(Thu)
+              </h3>
+              <p class="location light-black heading-small">
+                @Conference Hall, Main Campus, AIOU Islamabad
+              </p>
+            </div>
+          </div>
```
Comment (keroloussamy) [inline about.html]: - [ ] Kindly your header section on the About page doesn't follow the original design layout. You have to have a page title and a block of text and a line of text and after that bold text with an underline and all of them are centered.

Your design
image


Original design
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+        </div>
+      </header>
+    <section class="about-us flx-col-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="myLine text-center" />
+      <img src="./images/conference_photo.jpg" alt="about us community photo" class="family-photo" />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
```
Comment (keroloussamy) [inline about.html]: - [ ] Kindly here update the image path, so you can solve this issue.
- [ ] Kindly the image for this section is too big, kindly make it smaller and center it.
- [ ] Kindly the text is above the pic not under it.

Kindly fix these issues to match the layout of the design.

Your design
image

Original design
image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+          <div class="first-activity-content flx-col-center">
+            <h3>2019</h3>
+            <p>Robotics Conference of Automation</p>
+          </div>
+        </div>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <footer class="footer flx-col-center" id="contacts">
+        <h3 class="contacts light-black heading-medium">Contact</h3>
```
Comment (keroloussamy) [inline about.html]: - [ ] Kindly the Partners section exists on the mobile version kindly add it.
Original design
image

Your design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @for-ashraf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (omarsalem7) [reply index.html]: - [ ] As the previous reviewer mention please add some padding bottom to this paragraph and align it to left.
> your design 
image

File: README.md [hunk 8]
```diff
@@ -1 +1,46 @@
...
+
+## Project Description
```
Comment (omarsalem7) [inline README.md]: - [ ] Kindly add your demo and like for loom video as mentioned before.

File: about.html [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+        class="family-photo"
+      />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Id mollitia
+          magni enim ducimus repudiandae assumenda fugit dolorum doloribus
+          ratione dolor cum, fugiat consequatur quisquam quos deserunt amet illo
+          recusandae distinctio incidunt tenetur veritatis facilis excepturi
+          eum. Consectetur repellendus sunt, dolorum exercitationem maiores
+          optio architecto voluptates deserunt quia nemo quod magni.
```
Comment (omarsalem7) [inline about.html]: - [ ] As mentioned before, please reduce size of the image to be consistent with the design and put the text above the image.

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

- [ ] Consider also to add partner section in about page to be consistent with the design.
> Behance design
image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @for-ashraf, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @for-ashraf , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**